page-component-builder/src/components/ComponentPalette/index.js

```diff
@@ -6,14 +6,32 @@ import {
   Button,
   Card,
   Modal,
-  ToggleButton,
-  SectionHeading
+  SectionHeading,
+  TextInput,
+  ToggleButton
 } from '@contentful/forma-36-react-components';
 import './style.scss';
 
 const ComponentPalette = props => {
   const [isShown, toggleShown] = useState(false);
   const [selectedTags, setSelectedTags] = useState([]);
+  const [searchText, setSearchText] = useState([]);
```

Comment:
I think `useState` should start off with a default of `""` (blank string) instead of an array